logger/module.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 type loggerModule struct {
+	modules.Module
```

Comment:
We're no longer type checking that `var _ modules.Modules = &loggerModule{}`. Should we create a `RegistrableModule` for this use case?

Reply:
I wasn't really familiar with the logging implementation, I am getting my feet wet now :) I now understand that we don't need to register it because it's globally available and then within the `Start()` of each model we branch out.

This embed is not necessary anymore.

For consistency I have added 

```go
var _ modules.Module = &loggerModule{}
```